pyclibrary/c_parser.py

```diff
@@ -35,6 +35,91 @@
 __all__ = ['win_defs', 'CParser']
 
 
+def wrap_int(t):
+    logger.debug('wrap_int: {} {}'.format(t.dump(), type(t)))
+    t[0] = "CInt({})".format(int(t[0], 0))
```

Comment:
Why do we return a string from here? I would expect this function to return a CInt instance.

Reply:
I realise this should be part of a bigger rewrite, so no need to change in this PR.